src/estimate_parameters.cpp

```diff
@@ -354,5 +357,12 @@ uint32_t estimate_parameters(std::shared_ptr<pangenome::Graph> pangraph,
     for (auto& node : pangraph->nodes) {
```

Comment:
could we have a brief chat about what this is fdoing?

Reply:
Yep, what exactly? This `for` or the changes in this file?

Reply:
basically i dont understand what the code is doing (before or after your change)